extensions/amp-list/0.1/amp-list.js

```diff
@@ -552,19 +554,16 @@ export class AmpList extends AMP.BaseElement {
       scheduleNextPass();
       current.rejecter();
     };
-    if (this.ssrTemplateHelper_.isSupported()) {
-      const html = /** @type {string} */ (current.data);
```

Comment:
Seems like we're passing in a `string` to `renderTemplate()`, not `{html: <string>}` as it expects.

Reply:
We used too, but I modified ssrTemplate_ to return the JsonObject as opposed to just the html value of that object. Before it was `return response['html'];` and now `return response;`

So current.data is now a JSON object with the 'html' attribute in the case of SSR.